### ISSUE TITLE
Improvement: Improve general summarization pipeline

### ISSUE THREAD

**Issue by rmusser01** [rmusser01/tldw_server#45]
Title.

Whatever we can do to improve accuracy and speed of the summarization pipeline.

This issue can act as a tracker for ideas, approaches and information.

Articles:
https://github.com/voidism/Lookback-Lens
https://github.com/Vaibhavs10/optimise-my-whisper

Videos:
- [ ] https://www.youtube.com/watch?v=8OJC21T2SL4

Optimizing Whisper:
- [ ] https://github.com/Vaibhavs10/insanely-fast-whisper
- [ ] https://github.com/Vaibhavs10/optimise-my-whisper
- [ ] https://github.com/huggingface/distil-whisper
- [ ] https://huggingface.co/blog/asr-diarization

Flan-T5 Finetuning:
- [ ] https://huggingface.co/emonty777/QLoRA-Flan-T5-Small
- [ ] https://readmedium.com/simple-abstractive-text-summarization-with-pretrained-t5-text-to-text-transfer-transformer-10f6d602c426
- [ ] https://readmedium.com/mastering-ai-summarization-your-ultimate-productivity-hack-buildyourownai-cbd9e964dc65
- [ ] https://github.com/Erionis/Flan-T5-Summarization
- [ ] https://github.com/olonok69/LLM_Notebooks/blob/main/summarization/T5_base_Finetune_multi_news_summarization_v2_final.ipynb
- [ ] https://blog.eleuther.ai/pile-t5/
- [ ] https://www.philschmid.de/fine-tune-flan-t5
- [ ] https://arxiv.org/abs/2402.00841v2

Improving summarization approach:
- [ ] https://arxiv.org/abs/2309.04269

Unsorted Papers:
- [ ] https://arxiv.org/abs/2404.01588
- [ ] https://arxiv.org/abs/2406.11289
- [ ] https://aclanthology.org/2024.lrec-main.771/
- [ ] https://direct.mit.edu/tacl/article/doi/10.1162/tacl_a_00632/119276/Benchmarking-Large-Language-Models-for-News
- [ ] https://arxiv.org/pdf/2406.11289
- [ ] https://arxiv.org/abs/2309.02654
- [ ] https://arxiv.org/pdf/2004.04228
- [ ] https://huggingface.co/papers/2407.07071
- [ ] https://huggingface.co/papers/2407.05015
- [ ] https://huggingface.co/papers/2407.06071
- [ ] https://arxiv.org/abs/2309.15098
- [ ] https://lilianweng.github.io/posts/2024-07-07-hallucination/

https://github.com/AugmendTech/treeseg
https://huggingface.co/migtissera/HelixNet
https://arxiv.org/pdf/2407.18521
https://github.com/patched-codes/patchwork

**Comment by rmusser01** [comment]
Relevant links:
https://github.com/Vaibhavs10/insanely-fast-whisper
https://github.com/Vaibhavs10/optimise-my-whisper
https://github.com/huggingface/distil-whisper
https://huggingface.co/blog/asr-diarization

**Comment by rmusser01** [comment]
Flan-T5 Finetuning
https://huggingface.co/emonty777/QLoRA-Flan-T5-Small
https://readmedium.com/simple-abstractive-text-summarization-with-pretrained-t5-text-to-text-transfer-transformer-10f6d602c426
https://readmedium.com/mastering-ai-summarization-your-ultimate-productivity-hack-buildyourownai-cbd9e964dc65
https://github.com/Erionis/Flan-T5-Summarization
https://github.com/olonok69/LLM_Notebooks/blob/main/summarization/T5_base_Finetune_multi_news_summarization_v2_final.ipynb

**Comment by rmusser01** [comment]
https://arxiv.org/abs/2404.01588
https://arxiv.org/abs/2406.11289
https://aclanthology.org/2024.lrec-main.771.pdf

**Comment by rmusser01** [comment]
https://huggingface.co/google/gemma-7b-aps-it
https://huggingface.co/google/gemma-2b-aps-it
https://arxiv.org/abs/2406.19803
